### ISSUE TITLE
PT006: Could csv mode have a space between names?

### ISSUE THREAD

**Issue by notatallshaw** [astral-sh/ruff#12703]
I am working on adding [pytest rules for pip](https://github.com/pypa/pip/pull/12894). Turning on [PT006](https://docs.astral.sh/ruff/rules/pytest-parametrize-names-wrong-type/) with [lint.flake8-pytest-style.parametrize-names-type](https://docs.astral.sh/ruff/settings/#lint_flake8-pytest-style_parametrize-names-type) set to csv produces a slight styling difference on autofix of tests using tuple or list compared to pip's standard style.

It produces "[one,two,three](https://github.com/pypa/pip/pull/12894#discussion_r1703582965)" instead of "[one, two, three](https://github.com/pypa/pip/blob/3259c061fb36698339877bbf2cefb5e82815e812/tests/functional/test_install_extras.py#L197)". 

Would any of the following requests be considered?

* The autofix of tuple or list to csv converts to "one, two, three, found" not "one,two,three,four"
* This be the standard style so any csv string, e.g. "one,two  ,three,  four" gets nearly formatted to "one, two, three, four"
* An additional option to enforce either of these

**Comment by charliermarsh** [comment]
I'm personally fine with all of those changes, but I'm not super familiar with the norms here so would be nice to have others chime in :)

**Comment by MichaReiser** [comment]
Is there a way we could infer the preferred style, e.g. by preserving the existing spacing (similar to stylist inferring the line ending from the first use line ending)

**Comment by RonnyPfannschmidt** [comment]
I recommend adding the spaces for reading help

**Comment by Avasam** [comment]
"one, two, three" is also closer in style to what you'd get with list or tuple (ie: `["one", "two", "three"]`) from the formatter.

**Comment by charliermarsh** [comment]
I think this can be closed after #12853.
